### PR TITLE
Updates gusshawstewart slide caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @Aalim447
- [ ] @ErikMaranus
- [ ] @RodrigoTL
- [ ] @Sanmuhapriya259
- [ ] @daniel-thiele
- [ ] @dhanya-menon
- [ ] @fielpurvis
- [ ] @fvschult
- [ ] @gusshawstewart
- [ ] @guy-man
- [ ] @manuel-gaspar-maersk
- [ ] @sandipghosh2
- [ ] @veena-bhat
- [ ] @Athosonious
- [ ] @Devanshu1812
- [ ] @JGO074
- [ ] @Kath-Hall
- [ ] @PremKumar-K-Maersk
- [ ] @TKO087
- [ ] @amrita-ravindran
- [ ] @callofjudyy
- [ ] @gowrav-goyal
- [ ] @guillermo-garcia-maersk
- [ ] @jma493
- [ ] @mihai-buga
- [ ] @survesh-jha
- [ ] @yeshwant-jangid-maersk

### PR DESCRIPTION
**Pull request by gusshawstewart**
TL;DR  this PR is to update the caption of the gusshawstewart slide

This PR closes / fixes / resolves #2 

### Completed tasks
- created a branch
- updated the caption
- some other task

### Todo

- [ ] merge the PR
- [ ] delete the branch afterwards for good housekeeping

cc @ppremk

### CODE REVIEW

**Comment by gusshawstewart** [comment]
what do you think about doing it this way?

**Comment by RodrigoTL** [comment]
Looks good to go @gusshawstewart  👍

File: _slides/02-gusshawstewart.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Manchester United are quite rubbish at the moment!
```
Comment (TKO087) [inline _slides/02-gusshawstewart.md:6]: Great

**Review by TKO087 — APPROVED** [review]
